gap-analysis/index.md

```diff
@@ -268,13 +268,13 @@ Pros:
 Each of the solutions above have problems when applied to math. This has led the group to explore a new MathML-specific solution. The largest drawback to any MathML-specific solution is that it would expand any “MathML exists in its own world” criticism and wouldn’t leverage work done on the web to support existing web technologies, now or in the future. The generic advantage is that the solution would (mostly) not be held hostage hoping for changes to other specifications.
 
 ### Intent
-We have decided on using the term intent to refer to “the mathematical meaning conveyed by an author”, as applicable to presentation MathML. It is meant to offer a lightweight solution for partial annotation of an expression so that it can be spoken appropriately by AT. The intent syntax borrows some of the insights gained by Content MathML to encode the structure of mathematical operations (an “operator tree”), allowing for incremental narration and navigation that follows the argument structure of a formula.
+We have decided to use the term *intent* to refer to the mathematical meaning conveyed by an author using presentation MathML. It is meant to offer a lightweight solution for partial annotation of an expression so it can be spoken appropriately by AT. The intent syntax borrows concepts from Content MathML to encode the structure of mathematical operations (an “operator tree”), allowing for incremental narration and navigation that follows the argument structure of a formula.
 
-One proposal is to introduce a MathML-specific intent attribute, whose value encodes a functional expression composed using a mini language of literals (e.g. binomial-coefficient, factorial, sine, open-interval,...) as well as references into the appropriate descendent subtrees, when they act as conceptual arguments in a mathematical notation. .
 
+One proposal is to introduce a MathML-specific intent attribute, whose value encodes a functional expression composed using a vocabulary of function symbols (e.g. binomial-coefficient, factorial, sine, open-interval, ...) applied to references to descendent subtrees that act as conceptual arguments in a mathematical expression.
```

Comment:
>  I do not want to prejudge the means whereby we associate the symbols to the concepts, or to any specific natural-language wording for the concepts, all of which are still to be decided.

If this statement was true, you would have used a wildcard value such as `$concept1($arg1, $arg2)`, instead of "index". The values you tend to choose are skewing towards Content MathML, which is effectively instantiating a CAS-near framing over the approach. This is quite obvious really, just the way it is obvious that I'm advocating for a language-near, i.e. "encyclopedic" approach to the vocabulary of values. Maybe this is happening out of habit, but that's exactly why I'm here writing these comments from the other side.

If this is going to remain contentious, I would suggest we create two intent subsections, "CAS Intent" and "Natural Intent", where we can cross-compare the proposals. I do not feel comfortable with skewing so much towards pragmatic CMML in the examples.

A "coordinate" is a well-defined mathematical concept, and has no need to travel to the realm of "index", and `to select a component from a tuple`, which is something you do when you compute with an expression, not when you communicate it.